my-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css';
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.size = 9;
+    this.state = {
+
+      // wIsNext: true,
+      board: this.createGameBoard(this.size),
+
+    };
+  }
+
+  createGameBoard(size){
```

Comment:
When I see functions like `createSuchAndSuch`, they tend to stand out to me as a missed opportunity for an object:
```js
class GameBoard {
  constructor( size ) {
    this.size = size
    this.board = []
    this.initializeBoard()  // this would contain the board initialization loops below
  }
}
```
The `GameBoard` instance can now be created with:
```js
const board = new GameBoard( 9 );
```